src/pluginplay/printing/mermaid.cpp

```diff
@@ -14,7 +14,13 @@
  * limitations under the License.
  */
 
+#include "module_manager/module_manager_class.hpp"
```

Comment:
```suggestion
```
Since you included it in the header you shouldn't need to include it here too. That said including it again shouldn't hurt anything, but if you do want to include it again it should use angle brackets.